s3tbx-sentinel3-reader/src/main/java/org/esa/s3tbx/dataio/s3/slstr/SlstrLevel1B1kmProductReaderPlugIn.java

```diff
@@ -14,4 +15,13 @@ public SlstrLevel1B1kmProductReaderPlugIn() {
               "(S3.?_SL_1_RBT_.*(.SEN3)?)", "xfdumanifest", "L1c_Manifest", ".xml");
     }
 
+    @Override
+    public DecodeQualification getDecodeQualification(Object input) {
+        if(isInputValid(input)) {
+            return DecodeQualification.SUITABLE;
+        }else {
```

Comment:
Remove the spaces, please. Same thing in the 500m plugin.

Reply:
Which spaces do you mean? Actually, I would add some spaces.